src/components/Demo/buildEntities.tsx

```diff
@@ -1,10 +1,11 @@
-import type { BlockchainPolicyParameters } from '@nucypher/nucypher-ts'
 import React from 'react'
 import styled from 'styled-components'
 
 import { Button } from '../base/Button'
 import { ContentBlock } from '../base/base'
 import { Input, Label, TitleRow, CellTitle } from '../form/form'
+import type { ItDecConfig } from './tDec_and_conditions_demo'
```

Comment:
More of a style comment - seems like typescript typically uses camel case instead of snake case...?